etstool.py

```diff
@@ -109,6 +109,14 @@
     "docutils"
 }
 
+# Additional toolkit-independent dependencies for demo testing
+test_dependencies = {
+    "apptools",
+    "chaco",
+    "h5py",
+    "pytables",
+}
+
```

Comment:
These dependencies aren't traitsui runtime dependencies. On the other hand, I think the demo examples and the demo application could well have been a separate application-like distribution that can depend on other distributions more freely.

Reply:
Right.  On my to-do list has been to add a `demo` method to `etstool.py` and add those as dependencies there.  For now it is fine to add as testing dependencies.

Reply:
Thanks!